docs/building-on-etherlink/bridging.mdx

```diff
@@ -15,8 +15,33 @@ See the following pages for information about the kind of token that you want to
 Etherlink developers have created bridges for different situations.
 These bridges are trustless and permissionless; anyone can use them without restrictions or the intervention of a third party.
 
-Bridge name | Description | Mainnet link | Testnet link
---- | --- | --- | ---
-Canonical bridge | Bridges XTZ tokens from Tezos layer 1 to Etherlink and back | https://bridge.etherlink.com/ | https://testnet.bridge.etherlink.com/
-Wrapped asset bridge | Bridges assets from EVM chains to wrapped assets on Etherlink and back | https://www.etherlinkbridge.com/bridge | Not available on Etherlink Testnet
-Financial asset bridge | Bridges FA1.2 and FA2 [standards-compliant tokens](https://docs.tezos.com/architecture/tokens) from Tezos layer 1 to Etherlink and back | Coming soon | Coming soon
+<table class="customTableContainer">
+  <thead>
+    <tr>
+      <th>Bridge name</th>
+      <th>Description</th>
+      <th>Mainnet link</th>
+      <th>Testnet link</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Canonical bridge</td>
+      <td>Bridges XTZ tokens from Tezos layer 1 to Etherlink and back></td>
+      <td><a class="emphasize-advantage" href="https://bridge.etherlink.com/" target="_blank" rel="noopener noreferrer">Mainnet</a></td>
```

Comment:
I don't think this needs the `emphasize-advantage` class -- that's just for some places where we wanted to highlight otherwise plain text.